components/BtnContainer/BtnContainer.js

```diff
@@ -0,0 +1,20 @@
+import React, {PropTypes} from 'react';
+
+import css from './BtnContainer.css';
+
+export default class BtnContainer {
+  render() {
+    const {children, className, ...props} = this.props;
+
+    return (
+      <button {...props} className={[css.root, className].join(' ')}>
```

Comment:
Yeah, exactly that regarding style. Doesn't expect a class name as such, it's an optional addition if you want to style the `BtnContainer` for any particular reason.